oc_cdtapi/BitbucketAPI.py

```
@@ -0,0 +1,208 @@
+        # Form URL: {root}/rest/api/1.0/{req}
+        return posixpath.join(self.root, "rest", "api", "1.0", self.__req(req))
+    
+    def get_repo(self, project, repo_slug):
+        """
+        Get repository information
+        :param str project: project key
+        :param str repo_slug: repository slug
+        :return dict: repository information
+        """
+        logging.debug('Reached %s.get_repo', self.__class__.__name__)
+        logging.debug('project: {0}'.format(project))
```

Ruff (0.15.12)

[warning] Use implicit references for positional format fields

Remove explicit positional indices

(UP030)